.changeset/polite-apples-promise.md

```diff
@@ -0,0 +1,6 @@
+---
+"@itwin/itwinui-css": patch
+"@itwin/itwinui-react": patch
+---
+
+Fixes bug that caused the tabs `Wrapper` to change size dependent on which `Tab` was active.
```

Comment:
nit: Actually, you could consider splitting the changeset into two changesets with mostly the same content. The only difference would be using classnames in css changesets and exact component names in react changesets.

E.g. [react changeset](https://github.com/iTwin/iTwinUI/pull/1828/files#diff-1ae21bfe3b9b7e4ed39fbc59f54c851680020727241decbbc62dce30c2f3c88c), [css changeset](https://github.com/iTwin/iTwinUI/pull/1828/files#diff-65af2d9ac4e82ffa6b9332a961afe3bba98a8bf9de40e46e745cb84ce67a36bb).

Reply:
Done in latest commit. [(58ae4fd)](https://github.com/iTwin/iTwinUI/pull/1884/commits/58ae4fd2a674385dac3449c5e84510bb26e64c5d)